src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs

```diff
@@ -1400,7 +1400,7 @@ private void FindOwnerCompositeExecutable()
                 {
                     int oceOffset = GetOffset(section.RelativeVirtualAddress);
                     string ownerCompositeExecutable = Encoding.UTF8.GetString(Image, oceOffset, section.Size - 1); // exclude the zero terminator
-                    _ownerCompositeExecutable = ownerCompositeExecutable.ToEscapedString();
+                    _ownerCompositeExecutable = ownerCompositeExecutable.ToEscapedString().Replace("\"", ""); // remove double quotes
```

Comment:
Considering this is a self-inflicted wound, could we instead modify <code>ToEscapedString</code> to accept a flag suppressing the double-quoting rather than post-processing the constructed string?